packages/store/tests/issues/issue-933-selectors-causing-ticks.spec.ts

```diff
@@ -53,7 +53,7 @@ describe('Selectors within templates causing ticks (https://github.com/ngxs/stor
   class TestModule {}
 
   it(
-    'should run change detection whenever the selector emits after asynchronous action has been completed',
+    'should not run change detection whenever the selector emits after asynchronous action has been completed',
```

Comment:
Potentially better to say "should run change detection once for all selectors when acting action has been completed"